apps/ui/src/components/pages/application/utils/validatorKeys.ts

```diff
@@ -18,13 +18,21 @@
  */
 
 import {
-	type ApplicantDTO,
-	type ApplicationContentsResponse,
-	type InstitutionalRepDTO,
-	type InstitutionDTO,
-	type ProjectDTO,
+	AgreementDTO,
+	AppendicesDTO,
+	ApplicantDTO,
+	ApplicationContentsResponse,
+	InstitutionalRepDTO,
+	InstitutionDTO,
+	ProjectDTO,
```

Comment:
`type` keyword is removed here but added in https://github.com/Pan-Canadian-Genome-Library/daco/pull/338/files ?